Libplanet/Net/NetMQTransport.cs

```diff
@@ -396,6 +441,19 @@ public async Task RunAsync(CancellationToken cancellationToken)
             }
         }
 
+        /// <summary>
+        /// Finds and returns the <see cref="Peer"/> match with given <paramref name="target"/>.
```

Comment:
I suggested it because it will be represented that have the return value, in document and code level enoughly.

```suggestion
        /// Finds the <see cref="Peer"/> match with given <paramref name="target"/>.
```